container/values.yml

```diff
@@ -48,10 +51,14 @@ env:
   IMAGE: keg-test-consumer
   CONTAINER_NAME: keg-test-consumer
   VERSION: "0.0.1"
+  CHOKIDAR_USEPOLLING: 1
```

Comment:
`CHOKIDAR_USEPOLLING` is only needed for Taps, that use `keg-core` with webpack
Since the consumer doesn't, and the evf tap uses rollup in this context, you don't need to add it

Reply:
ohh ok got it. updated.
removed from `docker-compose.yml` and `values.yml`